examples/lb/listener/main.tf

```diff
@@ -0,0 +1,6 @@
+resource "vkcs_lb_listener" "listener" {
```

Comment:
keep resource string id equal to name w/ 'tf-example' suffix and underscores instead of dashes

Reply:
no reason to name resource "resource"

Reply:
> keep resource string id equal to name w/ 'tf-example' suffix and underscores instead of dashes

edited everywhere except loadbalancer i would not recommend naming all resources app-tf-example so i left name loadbalancer-tf-example and id app